### ISSUE TITLE
Test Scenarios shouldn't have dependencies on one another

### ISSUE THREAD

**Issue by mjhm** [mjhm/cucumber-api#8]
It looks like this Scenario is being used to seed the database for other scenarios. It would be better to have a separate explicit seeding step.  In most cases this would be in a "Background" step, so that it doesn't need to be copied into every step.

https://github.com/mjhm/cucumber-api/blob/master/example/testapp/apitest/features/basic.feature#L31

**Comment by allonsy** [comment]
I agree, I'll add a separate step definition that creates a seed database with dummy users
